config-controller/pkg/client/client.go

```diff
@@ -0,0 +1,301 @@
+package client
```

Comment:
Comment from @clickboo on https://github.com/stackrox/stackrox/pull/12562: policy client for naming? client is too generic and we have so many instances of client.go in the codebase already

Reply:
Would renaming this file to `policy.go` work for you?  Or are you wanting to rename the package as well?

Reply:
Aah there is a path, so leave it be I guess.